main/src/mill/modules/Jvm.scala

```diff
@@ -15,7 +17,35 @@ import mill.util.{Ctx, IO}
 import mill.util.Loose.Agg
 
 import scala.collection.mutable
+import scala.collection.JavaConverters._
 
+object Assembly {
```

Comment:
Is it right place to have this rules?
For people who want to define their own rules it will be:

```
import mill.modules.Assembly._
val myRules = Seq(Rule.Append("foo.conf"), Rule.Exclude("bar.txt"))
```